apps/dashboard/src/components/workflow-editor/steps/email/blocks/cards.tsx

```diff
@@ -0,0 +1,347 @@
+                    {
+                      type: 'variable',
+                      attrs: {
+                        id: 'payload.dueDate',
+                        label: null,
+                        fallback: null,
+                        required: false,
+                        aliasFor: null,
+                      },
+                    },
+                  ],
+                },
+                {
+                  type: 'paragraph',
+                  attrs: { textAlign: null, showIfKey: null },
+                  content: [
+                    {
+                      type: 'text',
+                      marks: [{ type: 'textStyle', attrs: { color: '' } }],
+                      text: 'If you add a payment method, the added payment method will be charged $250.00 or more every month, depending on usage.',
+                    },
+                  ],
+                },
+                {
+                  type: 'button',
+                  attrs: {
+                    text: 'Pay now',
+                    isTextVariable: false,
+                    url: '',
+                    isUrlVariable: false,
+                    alignment: 'center',
+                    variant: 'filled',
+                    borderRadius: 'smooth',
+                    buttonColor: '#f8f8f8',
+                    textColor: '#141313',
+                    showIfKey: null,
+                    width: '100%',
+                  },
+                },
+                {
+                  type: 'paragraph',
+                  attrs: { textAlign: null, showIfKey: null },
+                },
+              ],
+            },
+          ],
+        })
+        .run();
+    },
+  };
+};
+
+const createInformationCardWithLogo: (props: { track: ReturnType<typeof useTelemetry> }) => BlockItem = (props) => {
+  const { track } = props;
+
+  return {
+    title: 'Information card with logo',
+    description: 'Card: information card with logo',
+    searchTerms: ['logo', 'text', 'information', 'card'],
+    preview: '/images/email-editor/information-card-with-logo-preview.webp',
+    icon: <InformationCardWithLogo className="size-4" />,
+    command: ({ editor, range }) => {
+      track(TelemetryEvent.EMAIL_BLOCK_ADDED, {
+        type: 'header',
+      });
+
+      editor
+        .chain()
+        .deleteRange(range)
+        .insertContent({
+          type: 'section',
+          attrs: {
+            borderRadius: 6,
+            backgroundColor: '#f8f8f8',
+            align: 'left',
+            borderWidth: 2,
+            borderColor: '#f8f8f8',
+            paddingTop: 0,
+            paddingRight: 0,
+            paddingBottom: 0,
+            paddingLeft: 0,
+            marginTop: 0,
+            marginRight: 0,
+            marginBottom: 0,
+            marginLeft: 0,
+            showIfKey: null,
+          },
+          content: [
+            {
+              type: 'columns',
+              attrs: { showIfKey: null, gap: 0 },
```

Comment:
there should be more padding, but it doesn't work on a section, so we would have probably to play with the paddings in columns